code/modules/blob2/overmind/overmind.dm

```diff
@@ -25,7 +25,7 @@ var/list/overminds = list()
 
 	universal_understand = TRUE
 
-	var/list/has_langs = list(LANGUAGE_BLOB)
+	var/list/has_langs = list(LANGUAGE_ANIMAL)
```

Comment:
Blob might be unique enough to maybe have its own language? Idk that's subjective.

Reply:
We were talking about blob in the meeting and it was deemed such a rare thing to occur (We only used it for april fools), so it having an animal language isn't an issue.